doc/nrf/protocols/matter/getting_started/advanced_kconfigs.rst

```diff
@@ -172,3 +172,28 @@ Read Client functionality
 The Read Client functionality is used for reading attributes from another device in the Matter network.
 This functionality is disabled by default for Matter samples in the |NCS|, except for ones that need to read attributes from the bound devices, such as the :ref:`matter_light_switch_sample` and :ref:`matter_thermostat_sample` samples, and the :ref:`matter_bridge_app` application.
 Enable the feature if your device needs to be able to access attributes from a different device within the Matter network using, for example, bindings.
+
+Persistent storage
+==================
+
+The persistent storage module allows for the application data and configuration to survive the device reboot.
+|NCS| Matter applications use one generic Persistent Storage API, that can be enabled by the :kconfig:option:`CONFIG_NCS_SAMPLE_MATTER_PERSISTENT_STORAGE` Kconfig option.
+This API consists of methods with `Secure` and `NonSecure` prefixes, which handle secure (PSA Persistent Storage) and non-secure (raw Zephyr settings) storage operations accordingly.
```

Comment:
For me this text is clear, but I can see an area for confusion, because we call the newly created module Persistent Storage API (PSA), which is used altogether with ARM PSA (Platform Security Architecture). I think it is used correctly in all places, but maybe let's add PSA shortcut explanation as well?

Reply:
Heh, good catch.